### ISSUE TITLE
Current WebVTT base file naming convention conflicts with Islandora Batch base file naming convention

### ISSUE THREAD

**Issue by bradspry** [digitalutsc/islandora_solution_pack_oralhistories#1]
The currently required WebVTT file naming convention conflicts with Islandora Batch's base file naming convention.

Ideally, one would align the base file naming strategy with Islandora Batch, to enable command-line batch importing of oral histories.

OH123.xml
OH123.vtt
OH123.wav

**Comment by bradspry** [comment]
Attempting to batch import a zipfile with the following contents:
test.wav
test.xml
test_en.vtt

...results in a new object, with successful combination of test.wav and test.xml.

The presence of test_en.vtt however, causes the creation of a separate new object, with just test_en as the title of the MODS datastream.

If one renames test_en.vtt to test.vtt to align with the other files, islandora batch style, the transcript datastream is still not ingested during the batch process.

In summary, islandora_solution_pack_oralhistories is missing support for Islandora Batch import, but we are thankful for the developers who have brought the module to the very precipice of batch import support.

**Comment by Natkeeran** [comment]
Thank you reporting the issue.  

I assume that test.xml is the MODS stream.  OH module currently does not handle multiple vtts through batch!  (I am currently reviewing the support for using multi-lingual VTTs).  Thus, the batching is treating the test_en.vtt as a different object.

Also, Islandora Batch cannot handle xml data streams other than MODS or DC.  There is an alternate approach suggested here: https://github.com/mjordan/islandora_solution_pack_xml/tree/7.x/modules/islandora_simple_xml_batch.  However, we do to expect to support batch import in the short term.

**Comment by kstapelfeldt** [comment]
not for this release: keep on Roadmap though.

**Comment by kstapelfeldt** [comment]
To scope for next release - Islandora multi-importer may provide a workaround or use an approach similar to Mark's approach for supporting batch ingest in the XML solution pack. If so, we should document in the wiki documentation for those who wish to import batch oral histories.

**Comment by kstapelfeldt** [comment]
@MarcusBarnes to do

**Comment by kstapelfeldt** [comment]
Related to #72 and a general need to develop consensus around desired behaviour and what we wish to/can support, and how this is documented. 

Minimum: Ability to batch ingest video/audio files and at least related XML transcript datastreams
Interface batch ingest zip loader is and will remain unsupported in the Islandora 7 version unless it's something easy to do. 

Other issue: Are we providing a false affordance for people that suggests that interface ingest is possible? This needs to be reviewed with other content models that don't support batch. We could look at the XML solution pack and see what they are doing. We can also look at discimage and webarchive modules, which are part of the official release, and see if they are available under batch ingest.

**Comment by kstapelfeldt** [comment]
@kimpham54 to test existing discimage and webarchive batch ingest via interface and report.

**Comment by MarcusBarnes** [comment]
One can batch upload oral history objects using the Islandora Multi Importer.  This will be documented as part of resolving issue https://github.com/Islandora-Labs/islandora_solution_pack_oralhistories/issues/131.
